res/css/views/audio_messages/_PlaybackContainer.scss

```diff
@@ -45,18 +44,9 @@ limitations under the License.
         }
     }
 
-    .mx_Clock {
-        width: $font-42px; // we're not using a monospace font, so fake it
-        min-width: $font-42px; // force sensible layouts in awkward flexboxes (file panel, for example)
-        padding-right: 6px; // with the fixed width this ends up as a visual 8px most of the time, as intended.
-        padding-left: 8px; // isolate from recording circle / play control
-    }
-
     .mx_RecordingPlayback_timelineLayoutMiddle {
-        margin-left: 8px;
-        margin-right: 6px;
         position: relative;
-        display: inline-block;
+        display: flex;
```

Comment:
Nested flexboxes end up going poorly on Safari/Firefox, and I don't think we need it here.